ansible_collections/arista/avd/examples/l2ls-fabric/group_vars/DC1.yml

```diff
@@ -21,7 +21,7 @@ local_users:
     sha512_password: "$6$eucN5ngreuExDgwS$xnD7T8jO..GBDX0DUlp.hn.W7yW94xTjSanqgaQGBzPIhDAsyAl9N4oScHvOMvf07uVBFI4mKMxwdVEUVKgY/."
 
 # OOB Management network default gateway
-mgmt_gateway: 172.100.100.1
+mgmt_gateway: 172.16.100.1
```

Comment:
Need to fix documentation and looking at the doc, you probably need to fix the inventory.yml as well: https://avd.sh/en/stable/examples/l2ls-fabric/index.html#inventoryyml

Reply:
Done